index.html

```diff
@@ -0,0 +1,334 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta charset="ISO-8859-1">
+	<title>ROCA Observation Tool</title>
+	<link rel="stylesheet" href="obs.css">
+	<link rel="stylesheet" href="popupCSS.css">
+	<link rel="stylesheet" href="themify-icons.css">
+	<link rel="stylesheet" href="fonts/stylesheet.css">
+	<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+	<script type="text/javascript" src="obs.js"></script>
+	<script type="text/javascript">
+	function GetURLParameter(sParam)
+	{
+   		var sPageURL = window.location.search.substring(1);
+    	var sURLVariables = sPageURL.split('&');
+    	for (var i = 0; i < sURLVariables.length; i++) 
+    	{
+        	var sParameterName = sURLVariables[i].split('=');
+        	if (sParameterName[0] == sParam) 
+        	{
+            	return sParameterName[1];
+        	}
+    	}
+	}
+	var classroomname=(GetURLParameter('classroomname'));
+	$(document).ready(function(){
+		document.getElementById("classroomname").innerHTML = classroomname;
+
+  		var element = document.getElementById("classroom_mapping");
+  		element.classList.add(classroomname);
+	})
+	
+	</script>
+
+
+</head>
+
+
+<body onload="setAllDefaultValues()">
+	<!-- Data from forms set to invisible iframe: WILL NEED TO CHANGE THIS LATER -->
+	<iframe name="dump_data_frame" id="dump_data_frame"></iframe>
+
+	<div class="body_content">
+		<!-- BEGIN: left_side div -->
+		<!-- Contains classroom pic and majority of inputs -->
+		<div id="left_side">
+			<!-- BEGIN: top_left div -->
+			<!-- Contains back button and room name -->
+			<!-- <form autocomplete="off" id="student_events_form" target="dump_data_frame"></form> -->
+			<div id="top_left">
+				<div id="classroomname"><a onclick="reload()" style=" vertical-align: middle"><span id="back_arrow"
+							class="ti-arrow-left"></span></a>&nbsp;&nbsp;&nbsp;Thornton Hall E-Wing 303</div>
+
+				<!-- NEEDS: Pull From Database -->
+				<div class="class_section_input" id="s1">
+					<table style="width:auto; height:auto;">
+						<caption style="font-size: 40%; padding-top: 2%">Area 1</caption>
```

Comment:
I don't think we should hardcode individual areas because there may be more than 6 in a classroom. Maybe get rid of this section for now and we can have a separate ticket to work on this.